src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -268,6 +268,9 @@ public bool IsReturnStatement(SyntaxNode node)
         public bool IsStatement(SyntaxNode node)
             => node is StatementSyntax;
 
+        public bool IsExecutableStatement(SyntaxNode node)
+            => node is StatementSyntax;
```

Comment:
probably worth a comment explaining that this isn't a distinct concept in C#.

Reply:
➡️ Removed  IsStatement in dc69b593fd789990075e35344142defaf91c4f0f